packages/editor-ui/src/components/ModalDrawer.vue

```diff
@@ -23,6 +23,9 @@ export default Vue.extend({
 		name: {
 			type: String,
 		},
+		beforeClose: {
```

Comment:
@mutdmour This props was missing in this component. I added as in Modal.vue in order to achieve manually closing the Drawer.